start_scripts/start_gui.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+setup_bash_path='/home/nico/workspace/self_racing_rc_platform_ws/devel/setup.bash'
+session="self_racing_rc_platform-gui"
+
+cd ~/workspace/self_racing_rc_platform_ws/
```

Comment:
You can check in one of the other start scripts to see inspiration on how to not have the location hardcoded